internal/util/errors.go

```diff
@@ -39,27 +39,27 @@ var (
 	ErrMissingConfigForMonitor = errors.New("missing configuration of cluster ID for monitor")
 )
 
-type errorPair struct {
+type pairError struct {
```

Comment:
I do not think this is ever used, delete it completely?
Probably a left-over from before the project started to use `%w`.

Reply:
we are using `JoinErrors` which is a method of  `pairError` in many places.

Reply:
right, I guess that could be replaced as well, but no need to do this in this PR